pkg/multitenant/tenantcostmodel/settings.go

```diff
@@ -139,6 +141,14 @@ func validateRegionalCostMultiplierTableSetting(values *settings.Values, tableSt
 	return err
 }
 
+func validateEstimatedCPUSetting(values *settings.Values, jsonStr string) error {
```

Comment:
validateEstimatedCPUSetting should validate all of the slice pairs have the same length. Otherwise we could end up with an exciting misconfiguration that panics when a tenant crosses a certain batch size or hits a certain load threshold.

Reply:
Done.